src/components/CommandBar/CommandBarSelectionInput.tsx

```diff
@@ -12,13 +12,13 @@ import { modelingMachine } from 'machines/modelingMachine'
 import { useEffect, useMemo, useRef, useState } from 'react'
 import { StateFrom } from 'xstate'
 
-const semanticEntityNames: { [key: string]: Array<Selection['type']> } = {
-  face: ['extrude-wall', 'start-cap', 'end-cap'],
-  edge: ['edge', 'line', 'arc'],
-  point: ['point', 'line-end', 'line-mid'],
+const semanticEntityNames: { [key: string]: Array<Artifact['type']> } = {
+  face: ['wall', 'cap', 'solid2D'],
+  edge: ['segment', 'sweepEdge', 'edgeCutEdge'],
+  point: [],
```

Comment:
Would be cool to have a diagram that showed what operation generated what semantic entities. 
I know we have the visualized artifact graph but some short hand like

`extrude -> wall, cap, ...` so you know the list of possible values.

Reply:
I wonder if the feature tree will end up being the best visualisation for this?